Assets/Game/Scenes/Room/Room.unity

```diff
@@ -284,25 +284,25 @@ Mesh:
     offset: 0
```

Comment:
So there shouldn't be any changes to the `Room.unity` file. You'll wanna revert those so no changes show up here (see me if you need help with that). Your change should go on the FloppyDisk prefab instead.